install.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# Install Unix dependencies
+sudo wget -q https://apt.thoughtbot.com/thoughtbot.gpg.key -O /etc/apt/trusted.gpg.d/thoughtbot.gpg
+echo "deb https://apt.thoughtbot.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/thoughtbot.list
+sudo apt-get update
+sudo apt-get install -y silversearcher-ag rcm tmux vim
+
+# Clone thoughtbot's dotfiles
+git clone git@github.com:thoughtbot/dotfiles.git ~/dotfiles
```

Comment:
Could doing a [shallow clone](https://www.git-scm.com/docs/git-clone#Documentation/git-clone.txt---depthltdepthgt) speed things up a little, given you likely don't need the history?

```suggestion
git clone --depth 1 git@github.com:thoughtbot/dotfiles.git ~/dotfiles
```

Reply:
Resolved in 09eb199